### DISCUSSION TITLE
Please welcome Yassine in wallabag team!

### DISCUSSION THREAD

**Discussion by nicosomb** [wallabag/wallabag#6868] [Announcements]
After months of working on wallabag core (he has in particular worked very hard on 2.6 release https://github.com/wallabag/wallabag/releases/tag/2.6.0), I'm very pleased to welcome @yguedidi in our team 🦘 🎉  ! 

Yassine will help us to improve our great software. He will be able also to review and merge pull requests from contributors. 

Welcome @yguedidi and congrats!

**Comment by yguedidi** [comment]
Thank you @nicosomb !!
It's a pleasure for me to join the team: Of course I'll do my best to continue contributing as many things as I can!
Already some ideas to make wallabag an even better product :)

**Reply by nicosomb** [reply]
wallabag, not Wallabag! Be careful, I can ban you for this typo 😂

**Reply by yguedidi** [reply]
Fixed! @nicosomb yguedidi requested you a review.

**Comment by Simounet** [comment]
Congratz and thanks a lot for all you're doing for wallabag!

**Reply by yguedidi** [reply]
Thanks!
